### DISCUSSION TITLE
Outlook stop convert after meet second table tag when I'm trying convert .html to .msg

### DISCUSSION THREAD

**Discussion by Gray-Ice** [mhammond/pywin32#2536] [General]
When I'm trying to convert .html to .msg, I found outlook only convert the first and the second table tag, then stop convert anything next.
This is the html code:
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Email Test</title>
</head>
<body>
<table>
    <tr>
        <td>This is table1</td>
    </tr>
</table>

<table>
    <tr>
        <td>This is table2</td>
    </tr>
</table>

<table>
    <tr>
        <td>This is table3</td>
    </tr>
</table>

<h1>Here is something...</h1>
</body>
</html>
```
This is the python code:
```python
from win32com import client as win32
import os

outlook = win32.Dispatch("outlook.application")
mail = outlook.CreateItem(0)
mail.Subject = "This is a subject"

with open(".\\mix.html", "r", encoding="utf-8") as f:
    html = f.read()

mail.HtmlBody = html
current_path = os.getcwd()

mail.SaveAs(current_path + "\\rst.msg")

```
This is the final file I see:
![problem](https://user-images.githubusercontent.com/57894187/138387041-d46d37b5-b6ed-458f-819b-b5564e870633.jpg)

python version: 3.9.7
pip list:
pywin32                   302 
pywin32-ctypes            0.2.0

**Comment by mhammond** [comment]
This is a support request - https://github.com/mhammond/pywin32#support (although really it's an Outlook support request - it has nothing to do with Python or pywin32)
